File: Python/Stok.py
from Komputer import Komputer

class Stok:
    def __init__(self, jumlah=0, komputer=None):
        self.jumlah = jumlah
        self.komputer = komputer if komputer else Komputer()

    def set_jumlah(self, jumlah):
        self.jumlah = jumlah

    def set_komputer(self, komputer):
        self.komputer = komputer

    def get_jumlah(self):
        return self.jumlah

    def get_komputer(self):
        return self.komputer

File: CPP/Stok.cpp
#pragma once
#include <iostream>
#include <string>
#include "Komputer.cpp"

using namespace std;

class Stok {
private:
    int jumlah;
    Komputer komputer;
public:
    Stok()
    {

    }

    Stok(int jumlah, Komputer komputer)
    {
        this->jumlah = jumlah;
        this->komputer = komputer;
    }


    void setJumlah(int jumlah)
    {
        this->jumlah = jumlah;
    }

    void setKomputer(Komputer Komponen)
    {
        this->komputer = komputer;
    }

    int getJumlah()
    {
        return jumlah;
    }

    Komputer getKomputer()
    {
        return this->komputer;
    }

    ~Stok()
    {

    }
};
